### ISSUE TITLE
gcn_plugin gives wrong loss and accuracy in case of binary-class target

### ISSUE THREAD

**Issue by youph** [data61/stellar-evaluation-plugins#17]
This issue is caused by sklearn.preprocessing.LabelBinarizer() that behaves differently in case of binary class labels than in case of multi-class labels, see https://stackoverflow.com/questions/31947140/sklearn-labelbinarizer-returns-vector-when-there-are-2-classes

**Comment by youph** [comment]
Fixed in https://github.com/data61/stellar-evaluation-plugins/commit/347fffada193fa426d95277bf67933fdec69d00e by creating MyLabelBinarizer class inherited from LabelBinarizer, as suggested in https://stackoverflow.com/questions/31947140/sklearn-labelbinarizer-returns-vector-when-there-are-2-classes
